inc/saiqosmap.h

```diff
@@ -77,6 +77,12 @@ typedef enum _sai_qos_map_type_t
     /** QOS Map to set traffic class and color to EXP */
     SAI_QOS_MAP_TYPE_TC_AND_COLOR_TO_MPLS_EXP = 0x0000000c,
 
+    /** QOS Map to set DSCP to Forwarding class */
```

Comment:
Where are the bindpoints for these maps defined? We need it at a switch, port, RIF or tunnel etc. Today, we need something like SAI_PORT_ATTR_QOS_DSCP_TO_TC_MAP and SAI_SWITCH_ATTR_QOS_MPLS_EXP_TO_TC_MAP

Reply:
Yes, should be switch and port. I will add it.